Branch-SDK/Branch-SDK/BranchEvent.m

```diff
@@ -161,9 +180,30 @@ - (void) setContentItems:(NSMutableArray<BranchUniversalObject *> *)contentItems
     }
 }
 
+// Objective-C does not have string enums, convert for the server
```

Comment:
See here how to make obj-c 'string enums'.  

https://github.com/BranchMetrics/ios-branch-deep-linking/blob/7276778cf3ca1ec6af376950a37eed5290186348/Branch.framework/Versions/A/Headers/BranchEvent.h#L15

Reply:
Removed misleading comment.  I was thinking string enums in other languages that have built in type enforcement.  If I were to use the obj-c string enum macro, I'd still have to validate.